EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -242,4 +305,111 @@ private void addSubmitButton() {
         final Button submitButton = new Button("submitButton");
         layoutForm.add(submitButton);
     }
+
+    private void addExportValuesButton() {
+        final AjaxLink exportButton = new AjaxLink<Void>("exportValuesButton") {
+            @Override
+            public void onClick(AjaxRequestTarget target) {
+                layoutForm.modelChanged();
+                final AJAXDownload jsonDownload = new AJAXDownload() {
+                    @Override
+                    protected IResourceStream getResourceStream() {
+                        Map<TypeTreeNode, String> inputsForExporter = new HashMap<>();
+                        for (AttributeInput input : attributeInputs) {
+                            inputsForExporter.put(input.getAttribute(), input.getInput());
+                        }
+                        final File json = JsonExporter.generateExportFileWithValues(selectedEventType, inputsForExporter, eventCount, scaleFactor, eventTimestamps);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 164 characters long line (which is greater than 150 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00103)